### PR TITLE
10.12.2 Release Notes

### PR DESCRIPTION
**Pull request by pako81**
10.12.2 Release Notes.

### CODE REVIEW

File: modules/ROOT/pages/server_release_notes.adoc [hunk 1]
```diff
@@ -40,9 +41,17 @@
...
+Dear ownCloud administrator, find below the changes and known issues in ownCloud Server 10.12.2 that need your attention. You can also read the {owncloud-server-changelog-url}[full ownCloud Server changelog] for further details on what has changed.
+
+=== Filter sensitive data in log for Session::loginInOwnCloud
+
+The `loginInOwnCloud` method contains sensitive data in the argument list and needed therefore to be added to the list of methods where sensitive parameters are being obfuscated. https://github.com/owncloud/core/pull/40792[#40792]
+
 == Changes in 10.12.1
 
-Dear ownCloud administrator, find below the changes and known issues in ownCloud Server 10.12.1 that need your attention. You can also read the {oc-changelog-url}[full ownCloud Server changelog] for further details on what has changed.
+Dear ownCloud administrator, find below the changes and known issues in ownCloud Server 10.12.1 that need your attention. You can also read the {owncloud-server-changelog-url}[full ownCloud Server changelog] for further details on what has changed.
```
Comment (mmattel) [inline modules/ROOT/pages/server_release_notes.adoc:54]:
```suggestion
Dear ownCloud administrator, find below the changes and known issues in ownCloud Server 10.12.1 that need your attention. You can also read the {oc-changelog-url}[full ownCloud Server changelog] for further details on what has changed.
```
There is no `owncloud-server-changelog-url` attribute, see `site.yml`

Comment (pako81) [reply modules/ROOT/pages/server_release_notes.adoc:54]: Before those changes links were not not correctly displayed. There seems to be `:owncloud-server-changelog-url: https://owncloud.com/changelog/server/` in modules/ROOT/pages/server_release_notes.adoc ?

Comment (pako81) [reply modules/ROOT/pages/server_release_notes.adoc:54]: Ah apparently links in the online release notes are displayed correctly. Will revert the changes.

**Comment by pako81** [comment]
Apparently https://github.com/owncloud/core/pull/40803 has been already merged into the 10.12.2 release branch but as a separate commit https://github.com/owncloud/core/commit/5c2fc08347234f80f303a2c25b8ff531b39b9f08.

Still I guess in the Release Notes we should point to https://github.com/owncloud/core/pull/40803 even if not merged yet? Going to update them.

**Comment by pako81** [comment]
No more items should go into the 10.12.2 Release Notes.

**Review by hodyroff — APPROVED** [review]

File: modules/ROOT/pages/server_release_notes.adoc [hunk 2]
```diff
@@ -40,6 +41,18 @@
 * xref:changes-in-8-0[Changes in 8.0]
 * xref:changes-in-7-0[Changes in 7.0]
 
+== Changes in 10.12.2
+
+Dear ownCloud administrator, find below the changes and known issues in ownCloud Server 10.12.2 that need your attention. You can also read the {oc-changelog-url}[full ownCloud Server changelog] for further details on what has changed.
+
+=== Disallow permissions to be upgraded via federated sharing
+
+Do not allow to set higher permissions on a federated share for a resource which was shared with limited permissions. https://github.com/owncloud/core/pull/40803[#40803]
```
Comment (phil-davis) [inline modules/ROOT/pages/server_release_notes.adoc:50]:
```suggestion
Do not allow setting higher permissions on a federated share for a resource which was shared with limited permissions. https://github.com/owncloud/core/pull/40803[#40803]
```

**Review by phil-davis — APPROVED** [review]

**Review by mmattel — APPROVED** [review]

**Review by jnweiger — APPROVED** [review]
LGTM, hope that {oc-changelog-url} is a thing.

**Pull request merged**